src/pages/chatRoom.tsx

```diff
@@ -0,0 +1,169 @@
+import React, { useState, useEffect, useRef } from 'react';
+import NavBar from '../components/navBar';
+import DateDivider from '../components/dateDivider';
+import MyMessageBox from '../components/myMessageBox';
+import OtherMessageBox from '../components/otherMessageBox';
+import PlusButton from '../components/plusButton';
+import ChatHeader from '../components/chatHeader';
+import messagesData from '../data/chatData.json';
+import ProfileButton from '../components/profileButton';
```

Comment:
craco설치하면 절대경로 아래와 같이 심플하게 작성할 수 있답니다.
상대경로 쓰면 이후 파일위치 옮길때마다 수정하기가 상당히 귀찮은데, 이거쓰니까 좋더라구요

저희 과제란 밑에 참고 자료랑 아래의 자료 참고하면 반영할 수 있을 거예요~!
[절대경로 설치](https://tesseractjh.tistory.com/232)

```suggestion
import NavBar from '@components/navBar'
import DateDivider from '@components/dateDivider';
import MyMessageBox from '@components/myMessageBox';
import OtherMessageBox from '@components/otherMessageBox';
import PlusButton from '@components/plusButton';
import ChatHeader from '@components/chatHeader';
import messagesData from '@data/chatData.json';
import ProfileButton from '@components/profileButton';
```

Reply:
ㅠㅠ 갑자기 끼어들어 죄송함니다만... 나도 이거 깔았었는데 갑자기 이유를알수없는에러가... ㅜㅜㅜㅜ 나서 삭제햇어요... 다시해보려고요... 너무좋은것가튼데 지금 경로가 '../../../../' 이 난리...